content/2016-04-05-benchmarks.rst

```diff
@@ -0,0 +1,185 @@
+Compiler Flags
+##############
+
+:date: 2016-04-05
+:category: benchmark
+:lang: en
+:authors: Lightjohn
+:summary: Let's re-run an existing benchmark with pythran 
+
+The benchmark
+=================
+
+First there was a paper [0]_, 
+in the paper there was a github [1]_, 
+in the github [1]_ some benchmarks. 
+In my case I wanted to re-run the julia code because the language is changing quickly and so may do better. 
+But the day before I discovered `Pythran` so why not test both ?
+
+And so let's re-run two benchmarks: C++, Julia and add a new one, Pythran.
```

Comment:
maybe make the C++ and julia benchmark clickable so that the user can quickly see them?